README.md

```diff
@@ -1,141 +1,146 @@
+### Appeals
 
-Once the dispute is initialized and the jury formed, the adjudication process is broken up into four time windows, the duration of each can be different depending on the specific court being used:
+Each dispute is subject to a maximum number of appeal rounds. Since each dispute and appeal round has a fixed duration, the max appeal rounds also determines the maximum amount of time before a final decision is reached.
 
- - **Evidence Submission**: a period when parties of an agreement can submit statements and evidence to the jury.
- - **Jury Ruling Period**: a period for jurors to review submitted evidence and submit a ruling. The ruling determines if the case should be (1) escalated for review, (2) dismissed, (3) have the Agreement's FINALIZATION parameter updated and/or (3) whether collateral should be reassigned between parties. 
- - **Ruling Review Period**: a period which is triggered if atleast one juror flags the case for review, the evidence and jurors rulings are reviewed by a second set of jurors selected from the Review Court. The review process is intended to punish jurors who do not self-report bribery or vote in such a way that is not coherent with the jurors code of conduct.
- - **Appeal Period**: a period after the court has provided a ruling, but before reputation is assigned to participating jurors.
-### 3.2.1 Jury Ruling
-The Jury Ruling mechanism operates as a schelling point game where jurors individually assess the evidence and make a ruling and have an incentive to rule along with the majority of other jurors.
+Appeals can be triggered after a dispute has been resolved with a preliminary ruling in favor of one outcome or the other. In order for an Appeal to occur both sides of the dispute must deposit additional collateral. If neither side deposits the required collateral to trigger the next appeal round, the preliminary ruling is finalized. If only one side deposits the required collateral the ruling is immediately finalized in their favor. If both sides deposit the required collateral then the appeal round is scheduled.
 
-Since a ruling must aggregate input from all jurors, the following procedure is used to create a jury ruling and reward participants.
+The amount of required collateral depends on the appeal round and is a multiple of the fees required to compensate selected jurors. In higher appeal rounds where more juror stake is selected to adjudicate the dispute, the base amount of collateral required will be higher. The amount of collateral required will be a multiple of this amount, 2x the base amount for the side which is reinforcing the preliminary ruling, and 3x the base amount for side which is appealing the preliminary ruling. These deposit do not need to be supplied by a single party, but can be crowdsourced. When a final ruling is reached, the collateral of the loosing side will be used to compensate jurors and reward funders of the opposing side proportional to their contribution.
```

Comment:
The exact numbers / ratio is somewhat arbitrary, the goal is to ensure that the side that is reinforcing the ruling has an advantage, so that people are encouraged not to appeal.

Reply:
Hm, not sure if this sounds fair though.